test/UnsafeTest/src_90/org/openj9/test/unsafe/MainTester.java

```diff
@@ -47,20 +47,26 @@
 		List<Object> result = new ArrayList<>();
 		result.add(new TestUnsafeAccess(scenario));
 		result.add(new TestUnsafeAccessVolatile(scenario));
+		result.add(new TestUnsafeAccessOpaque(scenario));
 		result.add(new TestUnsafeAccessOrdered(scenario));
+		result.add(new TestUnsafeAccessUnaligned(scenario));
 		result.add(new TestUnsafeSetMemory(scenario));
 		result.add(new TestUnsafeCopyMemory(scenario));
 		result.add(new TestUnsafePutGetAddress(scenario));
-		result.add(new TestCompareAndSwap(scenario));
+		result.add(new TestUnsafeCompareAndExchange(scenario));
+		result.add(new TestUnsafeCompareAndSet(scenario));
 		result.add(new TestUnsafeAllocateDirectByteBuffer(scenario));
+		result.add(new TestUnsafeGetAndOp(scenario));
 		return result.toArray();
 	}
 	
 	private static void compileClass() {
 		try {
-			String[] classNames = { "MainTester", "TestCompareAndSwap", "TestUnsafeAccess",
-					"TestUnsafeAccessVolatile", "UnsafeTestBase",
-					"TestUnsafeAllocateDirectByteBuffer" };
+			String[] classNames = { "MainTester", "TestUnsafeAccess", "TestUnsafeAccessOpaque",
+					"TestUnsafeAccessOrdered", "TestUnsafeAccessVolatile", "TestUnsafeAllocateDirectByteBuffer",
+					"TestUnsafeCopyMemory", "TestUnsafeGetAndOp", "TestUnsafePutGetAddress",
+					"TestUnsafeCompareAndExchange", "TestUnsafeCompareAndSet", "TestUnsafeSetMemory", "UnsafeTestBase",
+					"TestUnsafeAccessUnaligned" };
```

Comment:
This would not fail. This code was in place before my changes. @TianyuZuo is there a preferred error to be thrown for this case?

Reply:
These are migrated from old repos and they were kept as-is. In the old one, it compiled it and logged the error but didn't fail the tests. 
My initial thought was that this was a test setup process but not a test case, so not finding the class didn't fail the whole test process and continue running. But it may be worth of changing it to Assert.fail if the test cases are not properly initialized.